prov/verbs/src/verbs_info.c

```diff
@@ -751,12 +751,27 @@ static int fi_ibv_getifaddrs(struct dlist_entry *verbs_devs)
 	int ret, num_verbs_ifs = 0;
 	size_t iface_len = 0;
 	int exact_match = 0;
+	unsigned int retry;
+
+	/* getifaddrs can fail when connecting the netlink socket. Try
+	 * again as this is a temporary error. After the 5th retry,
+	 * sleep a bit as well in case the host is really busy. */
+	retry = 10;
+	while (retry) {
+		ret = getifaddrs(&ifaddr);
+		if (ret == 0 || errno != ECONNREFUSED)
+			break;
+
+		retry --;
+
+		if (retry < 5)
```

Comment:
The check here is backwards from the comment above.  We could just always call:
```
usleep((1 << retry) * 1000);
```
for an exponential backoff.  This would require changing retry to count forward.

Reply:
Retry starts at 10 and decreases. So not backwards. Maybe misnamed though :)

I'll use the exponential stuff. In my testing, the second try always succeeded, so may be I'll use the exponentiation after 2 or 3 tries.